kubernetes/charts/weblogic-operator/templates/_operator-webhook-external-svc.tpl

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+{{- define "operator.operatorWebhookExternalService" }}
+{{- if .remoteDebugNodePortEnabled }}
```

Comment:
Do we need the if to test for both .remoteDebugNodePortEnabled and not operatorOnly?

Reply:
Yes, we do. Thanks. I have added check to for both `external-weblogic-operator-webhook-svc` and `weblogic-operator-webhook-svc` services for webhook.